src/Demo.tsx

```diff
@@ -148,10 +148,10 @@ export default class Demo extends React.Component<{}, DemoState> {
       <div id="demo">
         <Header text="Welcome to the demo page!" imgSrc="./logo.png" logOut="#" />
         <div className="list">
-          <Panel headerText="The default panel" content={panelBody} />
-          <Panel headerText="Success panel" style="success" content={panelBody} />
-          <Panel headerText="Warning panel" style="warning" content={panelBody} />
-          <Panel headerText="Danger panel" style="danger" content={panelBody} />
+          <Panel id="default" headerText="The default panel" content={panelBody} />
+          <Panel id="success" headerText="Success panel" style="success" content={panelBody} />
+          <Panel id="warning" headerText="Warning panel" style="warning" content={panelBody} />
+          <Panel id="danger" headerText="Danger panel" style="danger" content={panelBody} />
           <hr></hr>
```

Comment:
Don't the IDs need to be unique?  There are plenty of situations where you would have more than one of the same type of panel (e.g. the big list of success panels in our registry interface)...

Reply:
That's right but this is just a demo page and each `Panel` id on this page _is_ unique. The implementation for setting unique IDs in the registry admin is handled in a different PR.